sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
```

Comment:
Looks like they're all quotes and `'` is missing: https://www.sqlite.org/lang_keywords.html

So we should parse those as quotes and use the unquoted name.